bool online
int8 Zoom_Ratio
float64 Pitch_deg
float64 Yaw_deg